### PR TITLE
feat: Remove high-contrast header in table

### PR DESCRIPTION
**Pull request by fralongo**
### How has this been tested?

<details>
   <summary>Review checklist</summary>

_The following items are to be evaluated by the author(s) and the reviewer(s)._

#### Correctness

- _Changes include appropriate documentation updates._
- _Changes are backward-compatible if not indicated, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#public-apis)._
- _Changes do not include unsupported browser features, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#browsers-support)._
- _Changes were manually tested for accessibility, see [accessibility guidelines](https://cloudscape.design/foundation/core-principles/accessibility/)._

#### Security

- _If the code handles URLs: all URLs are validated through [the `checkSafeUrl` function](https://github.com/cloudscape-design/components/blob/main/src/internal/utils/check-safe-url.ts)._

#### Testing

- _Changes are covered with new/existing unit tests?_
- _Changes are covered with new/existing integration tests?_
</details>

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by michaeldowseza — APPROVED** [review]

**Review by michaeldowseza — APPROVED** [review]

File: src/wizard/internal.tsx [hunk 1]
```diff
@@ -170,14 +163,9 @@ export default function InternalWizard({
...
-          className={clsx(
-            styles.form,
-            isVisualRefresh && styles.refresh,
-            smallContainer && styles['small-container'],
-            shouldRemoveHighContrastHeader() && styles['remove-high-contrast-header']
-          )}
+          className={clsx(styles.form, isVisualRefresh && styles.refresh, smallContainer && styles['small-container'])}
         >
-          {isVisualRefresh && <div className={clsx(styles.background, getContentHeaderClassName())} />}
+          {isVisualRefresh && <div className={clsx(styles.background)} />}
```
Comment (michaeldowseza) [inline src/wizard/internal.tsx:168]: Non blocking: We don't need clsx here anymore

File: src/wizard/wizard-form-header.tsx [hunk 2]
```diff
@@ -17,22 +15,8 @@ export default function WizardFormHeader({ children, isVisualRefresh }: WizardFo
...
-      <div
-        className={clsx(
-          styles['form-header-content'],
-          shouldRemoveHighContrastHeader() && styles['remove-high-contrast-header']
-        )}
-      >
-        {children}
-      </div>
+    <div className={clsx(styles['form-header'], isVisualRefresh && styles['form-header-refresh'])} ref={overlapElement}>
+      <div className={clsx(styles['form-header-content'])}>{children}</div>
```
Comment (michaeldowseza) [inline src/wizard/wizard-form-header.tsx:19]: Non-blocking: We don't need clsx here anymore

**Review by michaeldowseza — APPROVED** [review]
Left some non-blocking comments.

**Comment by fralongo** [comment]
Merged with https://github.com/cloudscape-design/components/pull/2297

**Pull request closed without merging**